index.html

```diff
@@ -74,12 +74,21 @@ <h1 class="page-title">Daily Prebuilt demo</h1>
             <button
               id="join-call"
               onclick="joinCall()"
-              class="start-button disabled-button"
+              class="button start-button disabled-button"
             >
               Start demo
             </button>
           </div>
         </div>
+        <div id="error-message" class="hide">
+          <h2 id="error-title" class="title error-title">Incorrect room URL</h2>
+          <p id="error-description" class="error-description">
+            Meeting link entered is invalid. Please update the room URL.
+          </p>
+          <button class="button white-button" onClick="location.reload()">
```

Comment:
I wonder if instead of reloading you want to just clear the error state and go back to the form? If the URL is invalid it may have just been a typo, so no reload necessary. (In this case, just "Try again" should work for the button.) I'll leave to you to decide, though!

Reply:
This is a great idea. I added, thank you!